lib/_http_incoming.js

```diff
@@ -148,129 +146,33 @@ function _addHeaderLines(headers, n) {
 // 'array' header list is taken from:
 // https://mxr.mozilla.org/mozilla/source/netwerk/protocol/http/src/nsHttpHeaderArray.cpp
```

Comment:
small housekeeping, mozilla moved their server stuff, the url is `https://dxr.mozilla.org/mozilla/source/netwerk/protocol/http/src/nsHttpHeaderArray.cpp` now

Reply:
Thanks! Fixed.